### PR TITLE
Fix mesh network bug that report 0 when netserver is not reachable.

### PR DESCRIPTION
**Pull request by yuyantingzero**

### CODE REVIEW

**Comment by yuyantingzero** [comment]
test this please.

File: perfkitbenchmarker/benchmarks/mesh_network_benchmark.py [hunk 1]
```diff
@@ -113,6 +114,8 @@ def RunNetperf(vm, benchmark_name, servers, result):
 
   match = re.findall(r'(\d+\.\d+)\s+\n', output)
   value = 0
+  if len(match) != (len(servers) - 1) * FLAGS.num_connections:
+    raise errors.Benchmarks.RunError('Netserver not started on remote vm.')
```
Comment (cmccoy) [inline perfkitbenchmarker/benchmarks/mesh_network_benchmark.py]: Is netserver failing to start the only possible cause of errors here?

Please include the expected number of measurements and the number found in the error message.

Comment (yuyantingzero) [inline perfkitbenchmarker/benchmarks/mesh_network_benchmark.py]: Thanks for point out. It is not accurate. Netserver not reachable should be a better error message.

File: perfkitbenchmarker/benchmarks/mesh_network_benchmark.py [hunk 2]
```diff
@@ -113,6 +114,10 @@ def RunNetperf(vm, benchmark_name, servers, result):
 
   match = re.findall(r'(\d+\.\d+)\s+\n', output)
   value = 0
+  if len(match) != (len(servers) - 1) * FLAGS.num_connections:
+    raise errors.Benchmarks.RunError(
+        'Netserver not reachable. Expecting %s results, getting %s.' %
```
Comment (cmccoy) [inline perfkitbenchmarker/benchmarks/mesh_network_benchmark.py]: Nit: getting -> got.

File: perfkitbenchmarker/benchmarks/mesh_network_benchmark.py [hunk 3]
```diff
@@ -113,6 +114,10 @@ def RunNetperf(vm, benchmark_name, servers, result):
 
   match = re.findall(r'(\d+\.\d+)\s+\n', output)
   value = 0
+  if len(match) != (len(servers) - 1) * FLAGS.num_connections:
```
Comment (cmccoy) [inline perfkitbenchmarker/benchmarks/mesh_network_benchmark.py]: Nit: introduce a local for ` (len(servers) - 1) * FLAGS.num_connections`.

**Comment by cmccoy** [comment]
Two more small things, otherwise LGTM.

**Pull request merged**